integrations/docker/images/chip-cert-bins/Dockerfile

```diff
@@ -12,187 +12,187 @@ ARG ZAP_VERSION=v2023.04.27-nightly
 # Ensure TARGETPLATFORM is set
 RUN case ${TARGETPLATFORM} in \
         "linux/amd64") \
-            echo "Building for linux/amd64" \
+        echo "Building for linux/amd64" \
```

Comment:
Why the whitespace changes?

Reply:
Some auto-formatter ran. Undone the formatting.